decidim-proposals/app/views/decidim/proposals/admin/proposals/index.html.erb

```diff
@@ -66,7 +66,7 @@
           </tr>
         </thead>
         <tbody>
-          <% proposals.each do |proposal| %>
+          <% proposals.to_a.uniq.each do |proposal| %>
```

Comment:
Shouldn't the `distinct: true` already handle that? What's the source of the issue, is it a `join` that duplicates proposals?

Reply:
As the documentation says, probably there would be enough. I can't reproduce the issue in localhost.... is it ok to have a db dump?

Reply:
I guess it's OK - let's talk about it via Telegram maybe!